mmx-tests/Makefile

```diff
@@ -0,0 +1,29 @@
+USE_XMM=
```

Comment:
What these tests used for? Is your local tool uploaded unintentionally or you want them to be reviewed as well?

Reply:
These are the tests I used to validate that the re-implementation works properly: I found it extremely important to have validation that my implementation works properly, because i certainly messed up a few times.

I did upload it intentionally, but I don't propose to commit as-is -- especially not in this location.

I definitely would like to commit something along these lines _somewhere_, however, so that future maintainers have the benefit of being able to test this functionality. Probably it belongs in the llvm-test-suite repository?

I'd love help on getting this into a submittable form.

Reply:
How about we move these old implementations in to a seperate file (or leave them where they are if you like) and rename them to xxxx_dept, so that we don't rely on old compilers? We can then remove them as well as the tests in the next release.